### PR TITLE
Document new syntax for overriding container threadpool configuration

### PR DESCRIPTION
**Pull request by bjorncs**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: documentation/performance/container-tuning.html [hunk 1]
```diff
@@ -16,20 +16,40 @@
...
+    and the amount of garbage produced per request.
+<p>
 <pre>
 &lt;container id="container" version="1.0"&gt;
+  ...
+  &lt;search&gt;
+    &lt;threadpool&gt;
+      &lt;max-threads&gt;500&lt;/max-threads&gt;
+      &lt;min-threads&gt;500&lt;/min-threads&gt;
+      &lt;queue-size&gt;0&lt;/queue-size&gt;
```
Comment (jonmv) [inline documentation/performance/container-tuning.html:40]: What is the queue size? (I know, but readers won't.)

**Review by jonmv — COMMENTED** [review]
One little comment, otherwise good!

File: documentation/performance/container-tuning.html (hunk 1 above)
Comment (bjorncs) [reply documentation/performance/container-tuning.html:44]: See new commit

**Comment by bjorncs** [comment]
@jonmv PTAL

**Review by jonmv — APPROVED** [review]
!

**Pull request merged**